pipeline/inputs/tail.md

```diff
@@ -34,7 +34,7 @@ The plugin supports the following configuration parameters:
 | Tag | Set a tag \(with regex-extract fields\) that will be placed on lines read. E.g. `kube.<namespace_name>.<pod_name>.<container_name>`. Note that "tag expansion" is supported: if the tag includes an asterisk \(\*\), that asterisk will be replaced with the absolute path of the monitored file \(also see [Workflow of Tail + Kubernetes Filter](../filters/kubernetes.md#workflow-of-tail-kubernetes-filter)\).                                                                                       |  |
 | Tag\_Regex | Set a regex to extract fields from the file name. E.g. `(?<pod_name>[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*)_(?<namespace_name>[^_]+)_(?<container_name>.+)-`                                                                                                                                                                                                                                                                                                                    |  |
 | Static\_Batch\_Size | Set the maximum number of bytes to process per iteration for the monitored static files (files that already exists upon Fluent Bit start).                                                                                                                                                                                                                                                                                                                                                               | 50M     |
-
+| File\_Cache\_Advise | Set the posix_fadvise in POSIX_FADV_DONTNEED mode. This will reduce the usage of the kernel file cache. This option is only available on Linux.                                                                                                                                                                                                                                                                                                                                                              | On     |
```

Comment:
By only available do you mean configuring the option causes a failure on non-Linux or just it does nothing?

Reply:
It just does nothing
https://github.com/fluent/fluent-bit/blob/8542fac9c29750818324fee1a6ce1a800e6c82c6/plugins/in_tail/tail_file.c#L1438-1445

Reply:
OK, maybe reword then to say `will be ignored if not on Linux`?

Reply:
done.